app/lib/primer/classify.rb

```diff
@@ -123,13 +123,13 @@ def call(classes: "", style: nil, **args)
       def validated_class_names(classes)
         return if classes.blank?
 
-        if ENV["RAILS_ENV"] == "development"
+        if ENV["RAILS_ENV"] == "development" && !ENV["PRIMER_WARNINGS_DISABLED"]
```

Comment:
could we use an engine config instead? Something similar to `silence_deprecations`

Reply:
🤔 we could, but I was hoping to make this an easy command line flag, IE: `PRIMER_WARNINGS_DISABLED=1 script/server`

Reply:
FWIW the use case here is:

"I have to fix a bug in dotcom, but I'm getting a deprecation error when looking at the page. I don't own the view, or the fifty places it's linked, so I guess I'll just go and comment out the error raise in `/vendor`" 😉.